project/src/services/api.js

```diff
@@ -23,14 +23,28 @@ export const createAPI = (onUnauthorized) => {
   const onFail = (err) => {
     const {response} = err;
 
-    if (response.status === HttpCode.UNAUTHORIZED) {
+    if (response !== undefined && response.status === HttpCode.UNAUTHORIZED) {
```

Comment:
Тоже есть более короткий способ, но не знаю, поддерживается ли он линтером Академии

```suggestion
    if (response?.status === HttpCode.UNAUTHORIZED) {
```